### PR TITLE
AS4 Upgrade

### PR DESCRIPTION
**Pull request by mabuyo**
Upgrades to use Apollo Server 4.

DO NOT MERGE until course content is also updated.

### CODE REVIEW

File: final/monolith/index.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
-const { ApolloServer, gql, AuthenticationError } = require('apollo-server');
+const { ApolloServer } = require('@apollo/server');
+const { startStandaloneServer } = require('@apollo/server/standalone');
 const { buildSubgraphSchema } = require('@apollo/subgraph');
 
 const { readFileSync } = require('fs');
 const axios = require('axios');
+const { gql } = require('graphql-tag');
```
Comment (mabuyo) [inline final/monolith/index.js:7]:
```suggestion
const gql = require('graphql-tag');
```

File: final/subgraph-accounts/index.js [hunk 2]
```diff
@@ -1,41 +1,59 @@
-const { ApolloServer, gql } = require('apollo-server');
+const { ApolloServer } = require('@apollo/server');
+const { startStandaloneServer } = require('@apollo/server/standalone');
 const { buildSubgraphSchema } = require('@apollo/subgraph');
+
 const { readFileSync } = require('fs');
 const axios = require('axios');
+const { gql } = require('graphql-tag');
```
Comment (mabuyo) [inline final/subgraph-accounts/index.js:7]:
```suggestion
const gql = require('graphql-tag');
```

File: final/subgraph-template/index.js [hunk 3]
```diff
@@ -1,39 +1,58 @@
-const { ApolloServer, gql } = require('apollo-server');
+const { ApolloServer } = require('@apollo/server');
+const { startStandaloneServer } = require('@apollo/server/standalone');
 const { buildSubgraphSchema } = require('@apollo/subgraph');
+
 const { readFileSync } = require('fs');
 const axios = require('axios');
+const { gql } = require('graphql-tag');
```
Comment (mabuyo) [inline final/subgraph-template/index.js:7]:
```suggestion
const gql = require('graphql-tag');
```

File: monolith/index.js [hunk 4]
```diff
@@ -1,6 +1,11 @@
-const { ApolloServer, gql, AuthenticationError } = require('apollo-server');
+const { ApolloServer } = require('@apollo/server');
+const { startStandaloneServer } = require('@apollo/server/standalone');
+
 const { readFileSync } = require('fs');
 const axios = require('axios');
+const { gql } = require('graphql-tag');
```
Comment (mabuyo) [inline monolith/index.js:6]:
```suggestion
const gql = require('graphql-tag');
```

File: subgraph-template/index.js [hunk 5]
```diff
@@ -1,39 +1,58 @@
-const { ApolloServer, gql } = require('apollo-server');
+const { ApolloServer } = require('@apollo/server');
+const { startStandaloneServer } = require('@apollo/server/standalone');
 const { buildSubgraphSchema } = require('@apollo/subgraph');
+
 const { readFileSync } = require('fs');
 const axios = require('axios');
+const { gql } = require('graphql-tag');
```
Comment (mabuyo) [inline subgraph-template/index.js:7]:
```suggestion
const gql = require('graphql-tag');
```

**Review by lizhennessy — APPROVED** [review]
✅ starter code works great (one fix needed in Lesson 9 in Odyssey)
✅ final folder code works great

**Review by lizhennessy — APPROVED** [review]

**Pull request merged**